doc/features_without_segmented_area.rst

```diff
@@ -0,0 +1,46 @@
+Features without segmented areas
+--------------------------------------
+
+Not all detected features have a segmented area associated with them. Here, we show two cases in which a detected feature might
+not have a segmented area associated with them (meaning that the mask file does not contain the ID of the feature of interest and `ncells` in the segmentation
+output dataframe results in 0 grid cells. )
+
+
+.. _Case 1
+======
+Case 1: Segmentation threshold 
+======
+
+If the segmentation threshold is lower (assuming `target='minimum'`) than the highest threshold specified in the Feature Detection (see :doc:`threshold_detection_parameters`) this could leave some features without a segmented area, simply because there are no values to be segmented.
+
+Consider for example the following data with 5 being the highest threshold specified for the Feature Detection (see :doc:`feature_detection_overview`):  
+
+	.. image:: images/features_without_segment.png
+            :width: 500 px
+
+If the segmentation threshold is larger than 5 (e.g.  `threshold = 6`), the segmented area contains all values >= 5 (still assuming `target='minimum'`), no matter if the detected feature has a threshold lower than 5 (upper panels) or if it is exactly equal to 5 and does not contain any features with lower thresholds inside (lower panels).
```

Comment:
I think you still need to replace `values >= 5` by `values <= 5`, since these are contained in the segment for a minimum.

Reply:
Sorry @snilsn! I misinterpreted your comment and thought it was about clarifying that the target was still `minimum` in the segmentation. I totally missed that but fixed it now!